docs/getting-started/installation/wpilib.md

```diff
@@ -0,0 +1,6 @@
+# Installing WPILib
+Luckily the installation process for WPILib isn't changed much (if at all) by the use of Kotlin in your projects.
+
+To install WPILib you can simply follow the guide contained on the WPILib wiki, with a few special instructions. You can find this guide [HERE](https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-2/wpilib-setup.html). Once you reach the page where it asks which install mode you'd like, make sure to select "Everything", and then on the next page select "Skip and don't use VS Code".
+
+If you follow the guide, and make sure to select those options, you're done installing WPILib, and can move on to the next step.
```

Comment:
This sentence is probably unnecessary. I think we _can_ assume they follow instructions that we give